### DISCUSSION TITLE
Beginners question: how to see if my backups are containing what they should contain?

### DISCUSSION THREAD

**Discussion by Taxicletter** [borgbase/vorta#2054] [FAQ] [answered]
Hi,
I use Vorta on Fedora KDE (Framework laptop). I have a paid subscription to BorgBase and have a repository configured.
Everything seems to work. But to know for sure, I would like to check for instance my "Music" directory in my home folder and see what's there in a previous Archive.
To do that, without downloading the complete folder and of course without really restoring anything, what would I do? Do I choose "mount" (I have a Dutch version, where it's called "aankoppelen")? I tried that, but because it seemed to take a lot of time, I stopped, because I feared I was downloading the Archive that way.
Can I maybe even view what's exactly stored in the online repository?

**Comment by jeop10** [comment]
Hello!

Disclaimer: I am new and im just trying to learn Vorta as well 

I think that what you are asking can be only be achieved by mounting an archive into an empty directory

Depending on the size of your archive and internet connection this of course can take more or less time (in my short experience is very fast, but also my repo is very small around 4GB)

Another idea is to do a diff between two archives, so that you can see what has changed, this assumes that you added the Music directory recently and not from the begging of the repo.

If you are comfortable with the terminal you can do `borg list` on an archive and get a list of all the files that are inside that archive

Example: `borg list ssh://example@borgbase.com:/repo::user-2016-02-15` (please do check the syntax)

Link to the documentation: https://borgbackup.readthedocs.io/en/stable/usage/list.html

If you do find a better way to achieve your goal please write it here so others can learn it too.

Regards

**Reply by Taxicletter** [reply, marked as answer]
Thank you jeop10!

I read the manual a little better (https://vorta.borgbase.com/usage/restore/). If you choose "Extract" it **doesn't** immediately **extract** the archive (like I initially feared). Instead, you get a window with your folder tree. There you can choose what  you really want to extract.

Using _borg list ssh..._ in the terminal, ends in a long list of numbers and I guess encrypted names, so that isn't very useful. You can add '--short', and then you only have the numbers. Like this:

```
framboise-2024-08-02-181931          Fri, 2024-08-02 18:19:32 [3c32d1374d87264a3fb5c2323b793a806ee3dec706052c57c127bd2889a706b5]
framboise-2024-08-04-223157          Sun, 2024-08-04 22:32:00 [fe43c10227eae3901d57519cfd0eff3b791afdc053c3a77148e6112607d6efcc]
framboise-2024-08-05-141724          Mon, 2024-08-05 14:17:25 [6a7c99035c635e7a5ddd179437c61c2f1956dc81ce8b95dbb0c1a33279bc283d]
framboise-2024-08-06-183032          Tue, 2024-08-06 18:30:34 [5b118f2a47eb0754bed550e363b54638c5ba24f9616e5082f9c20a403afaf59b]
jonathan@framboise:~$ borg list --short ssh://xyz/repo
Enter passphrase for key ssh://xyz/repo: 
framboise-2024-05-21-221151
framboise-2024-05-22-064637
framboise-2024-05-22-124713
framboise-2024-05-22-213352
framboise-2024-05-22-222947
```
The further info on https://borgbackup.readthedocs.io/en/stable/usage/list.html is to complicated, I don't want to learn that much code too accomplish what I need :-). But that isn't necessary, since by extracting, I could see all my files and folders are indeed archived.

Thanks again, by your help, I found the patience to dig a little deeper :-)

**Reply by jeop10** [reply]
Awesome! Glad you worked it out!
